model/src/main/kotlin/licenses/ResolvedLicenseInfo.kt

```diff
@@ -72,13 +72,17 @@ data class ResolvedLicenseInfo(
     fun effectiveLicense(licenseView: LicenseView, vararg licenseChoices: List<SpdxLicenseChoice>): SpdxExpression? {
         val resolvedLicenseInfo = filter(licenseView, filterSources = true)
 
-        return resolvedLicenseInfo.licenses.flatMap { resolvedLicense ->
+        val detectedLicense = resolvedLicenseInfo.licenses.flatMap { resolvedLicense ->
```

Comment:
Nit: These are not necessarily only "detected" licenses, or? So maybe better call this `resolvedLicenses`?

Reply:
Thinking about it more, maybe `resolvedLicenseExpression` would be even more fitting. But I'm fine with keeping it as-is now.